web/client/components/I18N/Localized.jsx

```diff
@@ -37,6 +37,7 @@ class Localized extends React.Component {
             if (typeof children === 'function') {
                 children = children();
             }
+            document.documentElement.setAttribute("lang", this.props.locale);
```

Comment:
It would be better to move this in `componentDidMount` and `componentDidUpdate` and remove it from the `render` method, it could be something like:

```
componentDidMount() {
    this.updateDocumentLangAttribute();
}

componentDidUpdate(prevProps) {
    if (this.props.locale !== prevProps.locale) {
        this.updateDocumentLangAttribute();
    }
}

updateDocumentLangAttribute() {
    if (document?.documentElement) {
        document.documentElement.setAttribute("lang", this.props.locale);
    }
}
```